stdlib/public/runtime/HeapObject.cpp

```diff
@@ -111,21 +111,36 @@ static HeapObject *_swift_tryRetain_(HeapObject *object)
 #ifdef SWIFT_STDLIB_OVERRIDABLE_RETAIN_RELEASE
 
 #define CALL_IMPL(name, args) do { \
-    void *fptr; \
-    memcpy(&fptr, (void *)&_ ## name, sizeof(fptr)); \
-    extern char _ ## name ## _as_char asm("__" #name "_"); \
-    fptr = __ptrauth_swift_runtime_function_entry_strip(fptr); \
-    if (SWIFT_UNLIKELY(fptr != &_ ## name ## _as_char)) \
-      return _ ## name args; \
-    return _ ## name ## _ args; \
+  void *fptr; \
+  memcpy(&fptr, (void *)&_ ## name, sizeof(fptr)); \
+  extern char _ ## name ## _as_char asm("__" #name "_"); \
```

Comment:
`fptr` and the `_as_char` thing can be removed from this macro now.

Reply:
oh so they can, thank you